petclinic-frontend/src/features/inventories/api/addInventory.ts

```diff
@@ -0,0 +1,16 @@
+import { Inventory } from '@/features/inventories/models/Inventory.ts';
+import axiosInstance from '@/shared/api/axiosInstance';
+
+export default async function addInventory(
+  inventoryType: Omit<Inventory, 'typeId'>
```

Comment:
On this line, shouldn't it be InventoryId instead of typeId? Could you explain why you used typeId?

Reply:
Yes, this was a mistake, I'll modify this and change it to an inventoryId. Thanks for pointing it out